<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendFilebox</class>
 <widget class="QWidget" name="sendFilebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#recvmsg_lab{
	padding: 12px; 
	color: rgb(255, 255, 255);
	border-radius: 10px;
	background-color: rgb(0,153,255);
}

QLabel#background_lab{
	background-color: rgb(0,153,255);
	border-radius: 10px;
	border: 1px solid rgb(0,153,255);
}

QWidget#widget:hover{
	border-radius: 10px;
	border: 10px solid rgb(172, 172, 172);
}

QLabel#fileimg_lab{
	border-radius: 4px;
	border: 2px solid rgb(0, 255, 127);
}

QProgressBar#progressBar{
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>202</width>
       <height>92</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>202</width>
       <height>92</height>
      </size>
     </property>
     <widget class="QLabel" name="background_lab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>90</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="fileimg_lab">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../image/Image.qrc">:/new/icons/icons/becurrent/file.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>文件信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <widget class="QLabel" name="filename_lab">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>15</y>
         <width>110</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>文件名称</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="filesize_lab">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>35</y>
         <width>110</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>文件大小</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>180</width>
        <height>10</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>10</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="headImg_lab">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
